config-sample.json

```diff
@@ -2,6 +2,9 @@
   "bridge": {
     "name": "Homebridge",
     "username": "CC:22:3D:E3:CE:30",
+    "manufacturer": "Homebridge.io",
+    "firmware": "0.4.51",
```

Comment:
The `firmware` value doesn't seem to be used anywhere?

Reply:
Ah I see you removed it. So it should probably be removed from this sample config as well?

Reply:
Just removed it.